packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -564,20 +560,20 @@ class OverflowMenu extends Component {
 
     return (
       <ClickListener onClickOutside={this.handleClickOutside}>
-        <button
+        <IconButton
           {...other}
```

Comment:
One piece here I'm curious about is if folks are going to want to control the positioning of the tooltip. I can imagine someone opening an issue showing the menu conflicting with the tooltip placement. 

Can they do that here via `...other`, or would a more explicit prop type be preferred? At the very least maybe an update to the `.mdx` docs stating how the tooltip position can be configured?

Reply:
@laurenmrice do we need to update the website or design specs to show that OverflowMenu now includes a tooltip via IconButton. For context, this was the recommendation from @mbgower in the root issue.

Reply:
yes in theory just passing in the `align` prop to the overflowMenu should forward that to the IconButton , i'm happy to add a blurb in the docs about how the tooltip alignment and any other IconButton props can be passed onto it or we could make an explicit prop, whichever seems more correct?

Reply:
Added doc section on how to customize tooltip alignment: https://deploy-preview-12977--carbon-components-react.netlify.app/?path=/docs/components-overflowmenu--default#customizing-the-tooltip